### PR TITLE
chore(dependencies):  add eslint-plugin-fxa

### LINKED ISSUE #216: Use "eslint-plugin-fxa" in this repository
We need to pick up the latest changes (Buffer fixes, spacing, etc)

Example: https://github.com/mozilla/fxa-content-server/blob/master/package.json#L90

### PR DESCRIPTION
**Pull request by morrme**
fixes #216

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
   },
   "devDependencies": {
     "eslint-config-fxa": "2.1.0",
```
Comment (vladikoff) [inline package.json]: This needs to removed

**Comment by vladikoff** [comment]
Rerun `npm run shrinkwrap` to fix up the dependencies

**Comment by morrme** [comment]
@vladikoff i removed the line and made the 2nd commit before your comment. However, `npm run shrinkwrap` fails for me. It does create an output file (npm-shrinkwrap.json) but fails when trying to execute npmshrink.

**Comment by vladikoff** [comment]
@seanmonstar any clue what happened with `TypeError: hex is not a function` in CI?

**Comment by seanmonstar** [comment]
Yep, looks like the new `Buffer.from` API wasn't backported until v4.5: https://nodejs.org/en/blog/release/v4.5.0/

**Comment by morrme** [comment]
@seanmonstar is there anything I can do to fix this?

**Comment by seanmonstar** [comment]
Oh interesting, looking at the logs more closely, downloading the new node version failed the checksum verification. I'll just restart the build, sounds like a network error.

**Comment by vladikoff** [comment]
Fixed in https://github.com/mozilla/fxa-auth-db-mysql/commit/1157bb2b2dba85f6f43d98e69a25005ee2f225be

**Pull request closed without merging**